File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer;
    
    if(s.length() > 6 || s.length() < 4 || s.length() == 5)
    {
        answer = false;
        return answer;
    }
    
    for(int i=0; i<s.length(); ++i)
    {
        if((s[i] >= 'a' && s[i] <= 'z') || (s[i] >= 'A' && s[i] <= 'Z'))
        {
            answer = false;
            return answer;
        }
    }
    answer = true;
    
    return answer;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cs
public class Solution {
    public bool solution(string s) {
        bool answer = false;
        
        int sLength = s.Length;
        if(sLength == 4 || sLength == 6)
        {
            bool isNum = true;
            for(int i = 0; i < sLength; ++i)
            {
                if((s[i] >= 'a' && s[i] <= 'z') || s[i] >= 'A' && s[i] <= 'Z')
                {
                    isNum = false;
                    break;
                }
            }
            if(isNum)
                answer = true;
        }
        
        return answer;
    }
}